ext/js/language/deinflector.js

```diff
@@ -77,6 +79,18 @@ export class Deinflector {
         return results;
     }
 
+    /**
+     *
```

Comment:
Extra line here

Reply:
Bitwise operations are inherently pretty hard to understand, can you write some comments now that we're at it? Maybe not necessarily in this PR